firebase_admin/_auth_utils.py

```diff
@@ -275,6 +277,135 @@ def validate_provider_ids(provider_ids, required=False):
         validate_provider_id(provider_id, True)
     return provider_ids
 
+
+def validate_mfa_config(mfa_config):
```

Comment:
Makes sense but since all validations are defined here, I thought it would be a good place to keep the function. Also, project and tenant each use it from here and we can extend it to more admin users if needed? Since this is more of a utility function for validation rather than part of the config itself?

Reply:
Done